packages/jsii-dotnet-runtime-test/test/Amazon.JSII.Runtime.IntegrationTests/ComplianceTests.cs

```diff
@@ -915,7 +915,7 @@ public void OptionalAndVariadicArgumentsTest()
         [Fact(DisplayName = Prefix + nameof(JsiiAgent))]
         public void JsiiAgent()
         {
-            Assert.Equal("DotNet/" + Environment.Version.ToString(), JsiiAgent_.JsiiAgent);
+            Assert.Equal("DotNet/" + Environment.Version.ToString() + "/.NETStandard,Version=v2.0/1.0.0.0", JsiiAgent_.JsiiAgent);
```

Comment:
What does it take for this to break? Does it need manual action (source change somewhere else)? Or can it change just by getting updated toolkit?

Reply:
An update to your local framework.

I could just test that it StartsWith("DotNet")?

Reply:
As an aside, were we not going to change to core 2.1 target?

Standard is great, except it makes it look like we support framework - and I don't even really want to give developers that option.